README.md

```diff
@@ -48,16 +51,17 @@ cp ninja <a_directory_in_your_path>
 ## Buildings
 
 This is the simple one.
+You need pkg-config to be installed
```

Comment:
All right. I suppose I should add it to debian_native_dyn and debian_native_static, not for the other environments?